src/pages/Registration.jsx

```diff
@@ -0,0 +1,174 @@
+  };
+
+  const handleEmer1EmailChange = (e) => {
+    setEmer1Email(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleEmer2NameChange = (e) => {
+    setEmer2Name(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleEmer2PhoneChange = (e) => {
+    setEmer2Phone(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleEmer2EmailChange = (e) => {
+    setEmer2Email(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleUserNameChange = (e) => {
+    setUserName(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handlePasswordChange = (e) => {
+    setPassword(e.target.value);
+    setItemData({ ...itemData, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setProfileLog([...profileLog, itemData]);
+    setItemData({});
+    // navigate("/home");
+  };
+
+  // console.log(dataProfile)
+
+  return (
+    <div className="base-container">
+      <p className="font-light w-screen mt-10 text-3xl text-dark-pink text-left pl-10">
+        Welcome
+      </p>
+      <p className="font-light w-screen text-xl text-dark-pink text-left pl-10">
+        Let us know more about you!
+      </p>
+      <Form
+        name={name}
+        age={age}
+        weight={weight}
+        bloodType={bloodType}
+        medicalConditions={medicalConditions}
+        allergies={allergies}
+        phone={phone}
+        email={email}
+        emer1Name={emer1Name}
+        emer1Phone={emer1Phone}
+        emer1Email={emer1Email}
+        emer2Name={emer2Name}
+        emer2Phone={emer2Phone}
+        emer2Email={emer2Email}
+        userName={userName}
+        password={password}
+        onNameChange={handleNameChange}
+        onAgeChange={handleAgeChange}
+        onWeightChange={handleWeightChange}
+        onBloodTypeChange={handleBloodTypeChange}
+        onMedConditionsChange={handleMedConditionsChange}
+        onAllergiesChange={handleAllergiesChange}
+        onPhoneChange={handlePhoneChange}
+        onEmailChange={handleEmailChange}
+        onEmer1NameChange={handleEmer1NameChange}
+        onEmer1PhoneChange={handleEmer1PhoneChange}
+        onEmer1EmailChange={handleEmer1EmailChange}
+        onEmer2NameChange={handleEmer2NameChange}
+        onEmer2PhoneChange={handleEmer2PhoneChange}
+        onEmer2EmailChange={handleEmer2EmailChange}
+        onUserNameChange={handleUserNameChange}
+        onPasswordChange={handlePasswordChange}
+      />
+      <button
+        onClick={handleSubmit}
+        className="text-xl p-3 m-4 w-60 bg-dark-pink drop-shadow-lg hover:drop-shadow-xl hover:bg-pink-700 rounded-xl"
+      >
+        <Link to="/home">Record</Link>
+      </button>
+      <Link to="/home">
+        <Button value="To Main" />
```

Comment:
I found this button really odd tbh. You can register, or just bypass registration. I think it should either be no login required, or login enforced. This here is an in-between. If this is for development purposes, let's either render it only in development environment, or remove it before we push to production app.